lib/tests/2wp.js

```diff
@@ -105,13 +106,54 @@ const execute = (description, getRskHost) => {
 
     });
 
+    it('should do a basic pegin v1 with the exact minimum value', async () => {
+
+      // Arrange
+  
+      const initial2wpBalances = await get2wpBalances(rskTxHelper, btcTxHelper);
+      const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
+      const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
+      const peginValueInSatoshis = minimumPeginValueInSatoshis;
+      const peginV1RskRecipientAddress = await rskTxHelper.newAccountWithSeed('');
+  
+      // Act
+  
+      const peginV1Data = [Buffer.from(createPeginV1TxData(peginV1RskRecipientAddress), 'hex')];
+  
+      const btcPeginTxHash = await sendPegin(rskTxHelper, btcTxHelper, senderRecipientInfo.btcSenderAddressInfo, Number(satoshisToBtc(peginValueInSatoshis)), peginV1Data);
+  
+      // Assert
+  
+      const expectedCountOfThisPeginUtxosInTheBridge = 1;
```

Comment:
maybe this could be the default value for the param in `ensurePeginIsRegistered`? What do you think?

Reply:
It is. I just wanted to express it explicitly here. I can remove it and use it like this for multiple utxos.

Reply:
Removed.